site/documentation/content/api/management/device-registry-v1.yaml

```diff
@@ -391,6 +391,11 @@ paths:
                                  "via": [
                                     "gw-1", "gw-4"
                                  ],
+                                 "status": {
+                                    "created": "2019-10-03T13:45:16+02:00"
+                                    "updated": "2020-05-04T10:11:12+02:00"
```

Comment:
are you sure that these strings follow the definition of the properties? IIRC the times must not contain a time zone offset but be in UTC, right?

Reply:
You are correct, i made a quick copypasta there. I've put the Z for UTC.